src/Bridge/Symfony/DependencyInjection/TranslationAdapterPhraseAppExtension.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Translation\PlatformAdapter\PhraseApp\Bridge\Symfony\DependencyInjection;
+
+use FAPI\PhraseApp\PhraseAppClient;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+use Translation\PlatformAdapter\PhraseApp\PhraseApp;
+
+/**
+ * @author Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ */
+class TranslationAdapterPhraseAppExtension extends Extension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        $configuration = new Configuration($container);
+        $config = $this->processConfiguration($configuration, $configs);
+
+        $apiDef = $container->register('php_translation.adapter.phrase_app.raw');
+        $apiDef->setClass(PhraseAppClient::class);
+        $apiDef->addArgument($config['token']);
+
+        $adapterDef = $container->register('php_translation.adapter.phrase_app');
```

Comment:
We are not using the HTTPlug client or message factories from the configuration. See (copy) the [Loco adapter](https://github.com/php-translation/loco-adapter/blob/master/src/Bridge/Symfony/DependencyInjection/TranslationAdapterLocoExtension.php#L47-L65) .